app/views/directives/edit-environment-from.html

```diff
@@ -62,7 +64,16 @@
         </div>
       </div>
 
-      <div class="form-group environment-from-input prefix">
+      <div class="environment-from-input prefix">
+        <div
```

Comment:
Ideally this is a `<label for="...">` for accessibility, but that's a bit tricky since you have to generate IDs. I'm OK leaving that for later.